### ISSUE TITLE
Create sample dataset for machine learning projects

### ISSUE THREAD

**Issue by titaniumbones** [edgi-govdata-archiving/web-monitoring#6]
If possible, we should create a sample dataset drawn from the analysis team's records (collected January 7 februrary). Then we can maybe start to implement some simple rules and see if they help us to identify significant changes on a larger scale.

**Comment by 9jadataninja** [comment]
Will the datasets will be generated from web archives by the analysis team? Are there other alternatives to generate data sets?

**Comment by titaniumbones** [comment]
@9jadataninja I think that's the best option we have; otherwise I don't know how we would train the ML algorithms. Do you have other ideas?

**Comment by danielballan** [comment]
I'm told that the monitoring team is in the middle of reorganization and QA process which will be complete in about a week. Once that's done it will be easier to dig into this. I chatted with @trinberg just now to get oriented. I'll incorporate more details about the problem in the README soon.

**Comment by Mr0grog** [comment]
I’m planning to try and tackle pulling down actual HTML for each version from Versionista at the start of this coming week, so that should finally provide us with content we actually need for this. (Unless “web archives by the analysis team” already includes the HTML content? If so, that would be helpful to know!)

**Comment by dcwalk** [comment]
I just want to flag the size @danielballan mentioned in https://github.com/edgi-govdata-archiving/web-monitoring-processing/issues/21:

> My understanding is that ~50 is generally the bare minimum for even playing around with the data, so boosting the sample-data folder to, 50-100 curated examples would be a start. To train something we could actually use, my rough sense is that ~thousands are needed, likely depending on how rare true positives are

**Comment by janakrajchadha** [comment]
Hello everyone!
I'm an undergraduate student and I plan to submit a proposal for GSoC 2017.
I've been working on Machine Learning for a while and have recently dived into Deep Learning.

From my understanding and experience with small problems and projects, I can safely say that a small amount of data will lead to overfitting and the model won't generalise well. As we may face changes with characteristics we haven't seen before, it is important that the model generalises well.

The initial task of manually creating a dataset is inevitable. 
However, I had tried a trick to reduce the effort of manually creating a training dataset while working on a classification problem last year, and I think it may work here as well.

I believe I need to post on slack for a more detailed discussion, am I right @dcwalk ?

**Comment by ambergman** [comment]
@Mr0grog - Definitely exciting that you're going to be pulling down the actual HTML from Versionista, it'll be really great to have that safe. And, sadly, the analysis team's records definitely don't include any HTML content - but at least that means we're not duplicating efforts, Rob!

@danielballan's summary of the monitoring team's reorganization is definitely accurate, but @trinberg and I will try to pull together a training set that makes sense, with good documentation, by the end of this week or this weekend

**Comment by weatherpattern** [comment]
Does the data we shared with Data Kind satisfy this issue?

**Comment by danielballan** [comment]
It's a step in the right direction, but I'd like to leave this open until we have a set of Change UUIDs with associated information (significant, not significant, other interesting context) which won't really be possible until v0.0.4 because it involves parsing data from the so-called "dictionary".

**Comment by Mr0grog** [comment]
Should we remove the priority label and put this in the 0.0.4 milestone, then?
